.pre-commit-config.yaml

```diff
@@ -0,0 +1,8 @@
+repos:
```

Comment:
reviewer comment: adding black to `pre-commit` so it auto formats for us to remove manual labor